examples/ppo/agent.py

```diff
@@ -0,0 +1,12 @@
+"""Agent's moves (forward pass of the network)."""
+
+import jax
+import numpy as onp
```

Comment:
unused

Reply:
Thanks for spotting, since https://github.com/google/flax/pull/470/commits/9ff33b97091cc74620563934d6d526b0248fa11f there should be no unused imports.